### PR TITLE
Refs #10538: Add capsule removal documentation

### PR DESCRIPTION
**Pull request by chris1984**
…cs for un-installation (via katello-remove?)

Updated the user guide for how to remove a capsule with the capsule script that will be pushed in another commit.

### CODE REVIEW

**Comment by ehelms** [comment]
@chris1984 thanks, I might suggest updating the commit message to something that reflects the actual change and more concisely like "Refs #10538: Add capsule removal documentation"

File: docs/user_guide/capsules/index.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ In the simplest use case, a user may only want to use the Default Capsule. Large
 
 See [Capsule Installation](/docs/{{ site.version }}/installation/capsule.html)
 
+## Removal
+
+To stop all services and remove all Katello and Foreman related packages, run the following command as root on the capsule:
+
+  * `capsule-remove`
```
Comment (ehelms) [inline docs/user_guide/capsules/index.md]: We could drop the '*' here as that makes it appear as a list and just do a code-block:

```
capsule-remove
````

**Comment by ehelms** [comment]
Do you mind updating the merge request title to match similar to the commit message?

**Comment by stbenjam** [comment]
[test]

**Comment by ehelms** [comment]
Can you squash the 2 commits? Looks good after that.

**Comment by ehelms** [comment]
ACK, thanks @chris1984

**Pull request merged**